src/components/ForecastDetails.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "../styles/ForecastDetails.css";
+
+function ForecastDetails({ forecast }) {
+  const { date, humidity, temperature, wind } = forecast;
+  const formattedDate = new Date(date).toDateString();
+
+  return (
+    <div className="forecast-details" data-testid="forecast-details">
+      <div className="forecast-details__date">{formattedDate}</div>
```

Comment:
Is there any reason why you are using __ in a lots of classNames?
I have just googled this and it is used in Python for something, not like im any expert, but I think this can be confusing, I would suggest to use forecastDetailsDate or only one underscore "forecast_details_date" or "forecast-details-date"